Description: Build System patch for package builds.
 TODO: Put a short summary on the line above and replace this paragraph
 with a longer explanation of this change. Complete the meta-information
 with other relevant fields (see below for details). To make it easier, the
 information below has been extracted from the changelog. Adjust it or drop
 it.
 .
 jamnntpd (1.0-1~1a) UNRELEASED; urgency=low
 .
   * Initial release. (Closes: #223905)
   * Test build
Author: Robert James Clay <jame@rocasa.us>
Bug-Debian: http://bugs.debian.org/223905

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- jamnntpd-1.0.orig/src/Makefile.linux
+++ jamnntpd-1.0/src/Makefile.linux
@@ -7,7 +7,6 @@ EXESUFFIX =
 
 CC = gcc $(DEFS) -Wall
 RM = rm -f
-STRIP = strip
 
 OBJS = main.o nntpserv.o os_linux.o sockio.o groups.o misc.o xlat.o allow.o login.o mime.o
 
@@ -15,11 +14,9 @@ targets: jamnntpd$(EXESUFFIX) makechs$(E
 
 makechs$(EXESUFFIX): makechs.c
 	$(CC) $(PLATFORMDEF) makechs.c -o makechs
-	$(STRIP) makechs$(EXESUFFIX)
 
 jamnntpd$(EXESUFFIX): $(OBJS)
 	$(CC) -o jamnntpd$(EXESUFFIX) $(OBJS) jamlib/jamlib.a -lpthread
-	$(STRIP) jamnntpd$(EXESUFFIX)
 
 nntpserv.o : nntpserv.c
 	$(CC) $(PLATFORMDEF) -c nntpserv.c -o nntpserv.o
--- jamnntpd-1.0.orig/src/jamlib/Makefile.linux
+++ jamnntpd-1.0/src/jamlib/Makefile.linux
@@ -1,4 +1,4 @@
-CC = gcc -D__LINUX__ -Wall
+LDFLAGS+= -D__LINUX__ -Wall
 AR = ar -ru
 RM = rm -f
 
@@ -8,22 +8,22 @@ jamlib.a : $(OBJS)
 	$(AR) jamlib.a $(OBJS)
 
 crc32.o:    crc32.c jam.h
-	$(CC) -c crc32.c -o crc32.o
+	$(CC) $(LDFLAGS) -c crc32.c -o crc32.o
 
 mbase.o:    mbase.c jam.h
-	$(CC) -c mbase.c -o mbase.o
+	$(CC) $(LDFLAGS) -c mbase.c -o mbase.o
 
 message.o:  message.c jam.h
-	$(CC) -c message.c -o message.o
+	$(CC) $(LDFLAGS) -c message.c -o message.o
 
 lastread.o: lastread.c jam.h
-	$(CC) -c lastread.c -o lastread.o
+	$(CC) $(LDFLAGS) -c lastread.c -o lastread.o
 
 subpack.o:  subpack.c jam.h
-	$(CC) -c subpack.c -o subpack.o
+	$(CC) $(LDFLAGS) -c subpack.c -o subpack.o
 
 structrw.o:  structrw.c jam.h
-	$(CC) -c structrw.c -o structrw.o
+	$(CC) $(LDFLAGS) -c structrw.c -o structrw.o
 
 clean:
 	$(RM) *.o *.a
